test/tests/components/ws-manager/prebuild_test.go

```diff
@@ -310,16 +346,57 @@ func TestOpenWorkspaceFromPrebuild(t *testing.T) {
 							},
 						}, &resp)
 						if err == nil && resp.ExitCode == 0 && strings.Trim(resp.Stdout, " \t\n") != "" {
-							checkPrebuildSuccess = true
+							checkPrebuildLog = true
 							break
 						}
 
-						// wait 3 seconds and check
-						time.Sleep(3 * time.Second)
+						// wait 6 seconds and check
```

Comment:
I think this comment is unnecessary because we can understand what doing this code is. Instead of that comment, it would be better **why** sleep needs here.

Reply:
updated!